setup.py

```diff
@@ -6,12 +6,19 @@
 setup(
     name='csghub-sdk',
     version='0.3.1',
-    packages=find_packages(),
     author="opencsg",
     author_email="contact@opencsg.com",
     long_description=long_description,
     long_description_content_type="text/markdown",
+    packages=find_packages(include="pycsghub*"),
+    include_package_data=True,
+    entry_points={
+        "console_scripts": [
+            "csghub=pycsghub.cli:app",
```

Comment:
command line name `csghub-cli` is better

Reply:
fixed.